requirements.txt

```diff
@@ -1,13 +1,13 @@
 pip
 boto3==1.4.4
-cc-plugin-imos>=1.1.1
+cc-plugin-imos>=1.1.3
 celery==4.0.2
-compliance-checker==2.3.1
+compliance-checker>=3.1.1
```

Comment:
This looks fine to me, assuming we reliably catch version problems/incompatibilities when doing the packaging.